situp-core/src/integrationTest/resources/pipeline.yml

```diff
@@ -0,0 +1,11 @@
+raw-pipeline:
+  source:
+    otel_trace_source:
+  processor:
```

Comment:
We should have service-map also part of CI. Maye be next PR.

Reply:
Yeah, I will add service map in the next PR